lib/element.js

```diff
@@ -161,6 +161,100 @@ ElementArrayFinder.prototype.all = function(locator) {
 };
 
 /**
+ * Get a slice of elements within the ElementArrayFinder.
+ * Negative indices are wrapped (i.e. -i means ith element from last)
+ * This does not actually retrieve the underlying element.
+ *
+ * Slice can be negative in which case we go backwards.
+ *
+ * Note that if step is positive and start > stop, an error is thrown
```

Comment:
It should match `Array.prototype.slice`, since Protractor is a javascript tool and not a python tool.

Also, the comment implies that `elems.slice(2,-1)` would be invalid, since `2 > -1`.  Just some rewording would be good.